app/src/main/java/ru/prometeydev/movie/ui/moviesdetails/MoviesDetailsViewModel.kt

```diff
@@ -0,0 +1,18 @@
+package ru.prometeydev.movie.ui.moviesdetails
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import ru.prometeydev.movie.data.Actor
+
+class MoviesDetailsViewModel : ViewModel() {
```

Comment:
I would not use parcelable for movie, but just pass movie ID - it will be inject into model over factory and model will load movie from repository and cache it. That would have sense for me. Othervise this model doesn't have much sense. All this code just to check if array is empty not serious.

Reply:
I had such an idea, but apparently I was in a hurry. Yes, I will remake.